invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides

```diff
@@ -22,6 +22,11 @@
       padding-top: 0;
     }
 
+    &:not(.unstackable) thead tr:not(.stackable-content) {
```

Comment:
what is the usage of the `.stackable-content` class ?

Reply:
It was to fix the bulk actions in the tablet, but I think that a merge avoided the reason it was included in the first place.